integration/oidc/authorization-code/README.md

```diff
@@ -0,0 +1,197 @@
+generated using the Razor engine. Bootstrap and jQuery are available client-side.
+It is based on the dotnet core template named `webapp`.
+
+# Setup
+Before beginning the setup of this example, you should now already have two (2)
+app registrations in your Azure AD that represents WorkflowGen's portal and
+WorkflowGen's GraphQL API. If you don't, you must finish the authentication
+section of the WorkflowGen for Azure guide.
+
+## Create a New App Registration in Azure AD (Azure cli)
+The following commands create the necessary application registration in
+Azure AD using the Azure cli commandline tools.
+
+**Note**: You need to be logged in the Azure cli. To do this, run `az login`.
+
+### Create the application registration
+```powershell
+# Replace `WorkflowGen GraphQL API` with the name of the registered WorkflowGen
+# GraphQL API application including the backticks if you named it something else.
+$webApiApp = & az ad app list `
+    --query '[?displayName == `WorkflowGen GraphQL API`] | [0]' `
+    | ConvertFrom-Json `
+    | Select-Object -First 1
+$password = [guid]::NewGuid().ToString()
+$webAppUrl = "http://localhost:8080"
+
+# Create the application registration.
+$webAppClient = & az ad app create `
+    --display-name "My aspnet core Web App" `
+    --homepage $webAppUrl `
+    --identifier-uris $webAppUrl `
+    --password $password `
+    --end-date '2299-12-31T11:59:59+00:00' `
+    --reply-urls "$webAppUrl/signin" `
+    | ConvertFrom-Json
+
+# Create a service principal for the registration
+& az ad sp create --id $webAppClient.appId
+
+# Add required permissions for the GraphQL API access
+& az ad app permission add `
+    --id $webAppClient.appId `
+    --api $webApiApp.appId `
+    --api-permissions "$($webApiApp.oauth2Permissions.id)=Scope"
+
+# Grant the required permissions for the GraphQL API Access
+& az ad app permission grant `
+    --api $webApiApp.appId `
+    --id $webAppClient.appId
+```
+
+You will need the following information for the example configuration:
+
+* The client id. This is the value of `$webAppclient.appId`.
+* The client secret. This is the value of `$password`.
+* The GraphQL API App Id URI. This is the value of `$webApiApp.identifierUris[0]`.
+* The Azure AD Directory ID. Run the following command to get it:
+
+    ```powershell
+    az account show | ConvertFrom-Json | ForEach-Object tenantId
+    ```
+
+## Create a New App Registration in Azure AD (Azure Web Portal)
+To represent this example application that wants to access WorkflowGen's
+GraphQL API, we need to register it to Azure AD and give it access to the API.
+This way, when requesting the GraphQL API resource in the OIDC protocol, you
+will get an access token that is a valid JWT and that can be transmitted to
+the API for authorization.
+
+Keep in mind that the user that will authenticate with this application must
+exists in WorkflowGen.
+
+1. In Azure AD, go to the `App Registrations` section.
```

Comment:
1. In Azure AD, go to the **App Registrations** section and click the **New application registration** button.